inference-engine/ie_bridges/python/src/openvino/inference_engine/CMakeLists.txt

```diff
@@ -61,13 +61,28 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
 endif()
 
 # perform copy
-add_custom_command(TARGET ${TARGET_NAME}
-    POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/__init__.py ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/__init__.py
-    COMMAND ${CMAKE_COMMAND} -E copy ${PYTHON_BRIDGE_SRC_ROOT}/requirements.txt ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/../../requirements.txt
-    COMMAND ${CMAKE_COMMAND} -E copy ${PYTHON_BRIDGE_SRC_ROOT}/requirements.txt ${PYTHON_BRIDGE_OUTPUT_DIRECTORY}/../../requirements.txt
-    COMMAND ${CMAKE_COMMAND} -E copy ${PYTHON_BRIDGE_SRC_ROOT}/src/openvino/__init__.py ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/../__init__.py
-)
+if(CMAKE_BUILD_TYPE STREQUAL "Debug" AND NOT ("${CMAKE_DEBUG_POSTFIX}" STREQUAL ""))
+    add_custom_command(TARGET ${TARGET_NAME}
+        POST_BUILD
+        # add debug postfix to import statement (CVS-29031)
+        COMMAND ${PYTHON_EXECUTABLE} ${PYTHON_BRIDGE_SRC_ROOT}/cmake/update_file_statement.py
+            -i ${CMAKE_CURRENT_SOURCE_DIR}/__init__.py
+            --match_string "from .ie_api"
+            --replace_string "from .ie_api${CMAKE_DEBUG_POSTFIX}"
+            -o ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/__init__.py
```

Comment:
can we create a common cmake function and use it in `inference_engine`, `offline_transformations`, etc?